packages/mui-material/src/Select/Select.js

```diff
@@ -2,252 +2,346 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import { deepmerge } from '@mui/utils';
+import { unstable_composeClasses as composeClasses } from '@mui/core';
 import SelectInput from './SelectInput';
-import formControlState from '../FormControl/formControlState';
-import useFormControl from '../FormControl/useFormControl';
-import ArrowDropDownIcon from '../internal/svg-icons/ArrowDropDown';
-import Input from '../Input';
-import NativeSelectInput from '../NativeSelect/NativeSelectInput';
-import FilledInput from '../FilledInput';
-import OutlinedInput from '../OutlinedInput';
-import useThemeProps from '../styles/useThemeProps';
-import useForkRef from '../utils/useForkRef';
+import formControlState from '@mui/material/FormControl/formControlState';
+import useFormControl from '@mui/material/FormControl/useFormControl';
+import ArrowDropDownIcon from '@mui/material/internal/svg-icons/ArrowDropDown';
+import Input from '@mui/material/Input';
+import NativeSelectInput from '@mui/material/NativeSelect/NativeSelectInput';
+import FilledInput from '@mui/material/FilledInput';
+import OutlinedInput from '@mui/material/OutlinedInput';
+import useThemeProps from '@mui/material/styles/useThemeProps';
+import useForkRef from '@mui/material/utils/useForkRef';
+import { getSelectUtilityClasses } from './selectClasses';
+import { jsx as _jsx } from 'react/jsx-runtime';
```

Comment:
It seems that you based your PR on transpiled code. Please add your changes on top of the original source file.

Reply:
updated :)